en/02_Developer_Guides/20_Deprecations/index.md

```diff
@@ -28,29 +28,29 @@ going to be removed and how they need to update their code.
 
 When deprecating a method:
 
-- Add the following docblock `@deprecated 1.2.3 Use anotherMethod() instead`
-- `Deprecation::notice('1.2.3', 'Use anotherMethod() instead');` to the top of the method
+- Add the following docblock `@deprecated 1.2.0 Use anotherMethod() instead`
+- `Deprecation::notice('1.2.0', 'Use anotherMethod() instead');` to the top of the method
```

Comment:
`1.2.3` would not be correct for us, since we only add these in minor releases not patches.
Not related but seems prudent to fix.